packages/jest-resolve/src/index.js

```diff
@@ -91,8 +91,8 @@ class Resolver {
 
   static findNodeModule(path: Path, options: FindNodeModuleConfig): ?Path {
     const resolver = options.resolver
-      /* $FlowFixMe */
-      ? require(options.resolver)
+      ? /* $FlowFixMe */
```

Comment:
Could you open an issue against prettier? We may be able to print it the way you want, we're already doing that for flow unions.